src/Controls/src/NuGet/README.md

```diff
@@ -0,0 +1,282 @@
+
+### Prerequisites
+
+- [.NET 10 SDK](https://dotnet.microsoft.com/download/dotnet/10.0) (or .NET 9 for previous versions)
+- Platform-specific tools:
+  - **Android**: Android SDK (installed via Visual Studio or Android Studio)
+  - **iOS/macOS**: Xcode (Mac required)
+  - **Windows**: Windows App SDK
+
+### Installation
+
+Install the .NET MAUI workload:
+
+```bash
+dotnet workload install maui
+```
+
+### Create a New Project
+
+Create a new .NET MAUI app using the CLI:
+
+```bash
+dotnet new maui -n MyMauiApp
+cd MyMauiApp
+```
+
+Or create with sample content including Community Toolkit and Syncfusion Toolkit:
+
+```bash
+dotnet new maui -n MyMauiApp -sc
+```
+
+### Run Your App
+
+Run on Android:
+```bash
+dotnet build -t:Run -f net10.0-android
+```
+
+Run on iOS (Mac only):
+```bash
+dotnet build -t:Run -f net10.0-ios
+```
+
+Run on Windows:
+```bash
+dotnet build -t:Run -f net10.0-windows
+```
+
+## 💡 Quick Start Example
+
+Here's a simple .NET MAUI page with various controls:
+
+### XAML (MainPage.xaml)
+
+```xml
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="MyMauiApp.MainPage"
+             BackgroundColor="{DynamicResource PageBackgroundColor}">
+
+    <ScrollView>
+        <VerticalStackLayout Spacing="25" Padding="30">
+            
+            <Image Source="dotnet_bot.png" 
+                   HeightRequest="185"
+                   Aspect="AspectFit" />
+            
+            <Label Text="Hello, .NET MAUI!"
+                   Style="{StaticResource Headline}"
+                   HorizontalOptions="Center" />
+            
+            <Label Text="Build native apps for mobile and desktop"
+                   HorizontalOptions="Center" />
+            
+            <Entry Placeholder="Enter your name"
+                   x:Name="NameEntry" />
+            
+            <Button Text="Click Me"
+                    Clicked="OnCounterClicked"
+                    HorizontalOptions="Fill" />
+            
+            <Label x:Name="CounterLabel"
+                   Text="Button not clicked yet"
+                   HorizontalOptions="Center" />
+                   
+        </VerticalStackLayout>
+    </ScrollView>
+
+</ContentPage>
+```
+
+### Code-Behind (MainPage.xaml.cs)
+
+```csharp
+namespace MyMauiApp;
+
+public partial class MainPage : ContentPage
+{
+    private int count = 0;
+
+    public MainPage()
+    {
+        InitializeComponent();
+    }
+
+    private void OnCounterClicked(object sender, EventArgs e)
+    {
+        count++;
+        
+        if (count == 1)
+            CounterLabel.Text = $"Hello {NameEntry.Text}! Button clicked {count} time";
+        else
+            CounterLabel.Text = $"Hello {NameEntry.Text}! Button clicked {count} times";
+
+        SemanticScreenReader.Announce(CounterLabel.Text);
+    }
+}
+```
+
+## 🎯 Key Features
```

Comment:
Lets also call out XAML Source Generation and the XAML simplifications: https://learn.microsoft.com/dotnet/maui/xaml/xamlc?view=net-maui-10.0